README.rst

```diff
@@ -37,51 +40,50 @@ Example:
 Installation
 -------------
 
-Install using pip
+Install using `pip`_
 
   .. code:: bash
 
      [sudo] pip install google-endpoints-api-management
 
+.. _`pip`: https://pip.pypa.io
+
 
 Python Versions
 ---------------
 
-google-endpoints-api-management is currently tested with Python 2.7.
+endpoints-management-python is currently tested with Python 2.7.
 
 
-Contributing
-------------
+Contributions
+-------------
 
 Contributions to this library are always welcome and highly encouraged.
 
 See the `CONTRIBUTING`_ documentation for more information on how to get started.
 
+.. _`CONTRIBUTING`: https://github.com/cloudendpoints/endpoints-management-python/blob/master/CONTRIBUTING.rst
+
 
 Versioning
 ----------
 
 This library follows `Semantic Versioning`_
 
-It is currently in major version zero (``0.y.z``), which means that anything may
-change at any time and the public API should not be considered stable.
+.. _`Semantic Versioning`: http://semver.org/
 
 
 Details
 -------
 
-For detailed documentation of the modules in google-endpoints-api-management, please watch `DOCUMENTATION`_.
+For detailed documentation of the modules in endpoints-management-python, please watch `DOCUMENTATION`_.
 
+.. _`DOCUMENTATION`: https://endpoints-management-python.readthedocs.org/
 
-License
--------
+
+Licensing
+---------
```

Comment:
ok, I've tried a different approach
- The License title is an implicit hyperlink
-  clashes with the _`LICENSE`, an explicit hyperlink

I'll change the explicit one.